src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -221,7 +222,9 @@ public Builder addMember(String name, CodeBlock codeBlock) {
      * depending on the given {@code value} object. Falls back to {@code "$L"} literal format if
      * the class of the given {@code value} object is not supported.
      */
-    private Builder addValue(String memberName, Object value) {
+    public Builder addMemberForValue(String memberName, Object value) {
```

Comment:
Regardless, I’m reluctant to expose a new API here. Wanna remove the new API so we can unblock the rest of this PR ?